### PR TITLE
Faraway/commit

### PR DESCRIPTION
**Pull request by 0xSense**
A faraway land a merge fairies works very hard to ensure as much code is added into a merge as possible and resolves all conflicts of the commit land.  Merge  fairies are a mythical creature that live in the shadows of production code.

### CODE REVIEW

File: project.godot [hunk 1]
```diff
@@ -11,7 +11,11 @@ config_version=5
 [application]
 
 config/name="Game-jam-project-name-unknown"
-run/main_scene="res://scenes/overworld/overworld.tscn"
+<<<<<<< HEAD
+run/main_scene="res://scenes/main/main.tscn"
+=======
```
Comment (alexandervenezia) [inline project.godot]: Looks like not all the merge statements were properly removed?

**Review by Spencer-Trivette — COMMENTED** [review]
It's hard to leave a proper review on mobile lol

File: scenes/main/main.tscn [hunk 2]
```diff
@@ -1,24 +1,33 @@
-[gd_scene load_steps=6 format=3 uid="uid://cipd5odixnpm8"]
-
-[ext_resource type="Script" path="res://scenes/main/main.cs" id="1_6ei66"]
-[ext_resource type="Script" path="res://scenes/main/CardTestNode.cs" id="3_k0m6s"]
-[ext_resource type="Script" path="res://data_resources/card_data/MasterDeck.cs" id="4_x5pg5"]
-[ext_resource type="Script" path="res://scenes/main/GeneralTestNode.cs" id="5_6bc1x"]
+[gd_scene load_steps=9 format=3 uid="uid://cipd5odixnpm8"]
 
+<<<<<<< HEAD
```
Comment (alexandervenezia) [inline scenes/main/main.tscn:3]: Looks like more conflict markers

**Review by alexandervenezia — APPROVED** [review]
Looks good aside from noted qualifications.

File: scenes/overworld/TestLevel.tscn [hunk 3]
```diff
@@ -0,0 +1,477 @@
+<<<<<<< HEAD
```
Comment (Perplexer-HNP) [inline scenes/overworld/TestLevel.tscn:1]: Conflict markers :) :)

**Review by Perplexer-HNP — APPROVED** [review]
Seems fine, couple conflict markers but hopefully they dont break anything

**Pull request closed without merging**